plasmapy/atomic/particle_class.py

```diff
@@ -12,17 +12,8 @@
 import plasmapy.utils.roman as roman
 from plasmapy.atomic.elements import _Elements, _PeriodicTable
 from plasmapy.atomic.isotopes import _Isotopes
-from plasmapy.utils import (
-    AtomicError,
-    AtomicWarning,
-    InvalidParticleError,
-    InvalidElementError,
-    InvalidIsotopeError,
-    InvalidIonError,
-    ChargeError,
-    MissingAtomicDataError,
-    MissingAtomicDataWarning,
-)
+from plasmapy.atomic.exceptions import AtomicError, MissingAtomicDataError, ChargeError, InvalidIonError, InvalidIsotopeError, InvalidElementError, \
+    InvalidParticleError, AtomicWarning, MissingAtomicDataWarning
```

Comment:
We could probably use parentheses here and break this up into multiple lines again.  I'm curious if this was a relic of doing this change automatically.